adapter/pubsub/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+project(PubSubAdapter VERSION 0.1.0)
+
+set(HERMES_PUBSUB_ADAPTER_DIR ${HERMES_ADAPTER_DIR}/pubsub)
+
+set(PUBSUB_ADAPTER_PUBLIC_HEADER pubsub.h)
+set(PUBSUB_ADAPTER_PRIVATE_HEADER ${CMAKE_CURRENT_SOURCE_DIR}/metadata_manager.h
+        ${CMAKE_CURRENT_SOURCE_DIR}/datastructures.h
+        ${HERMES_ADAPTER_DIR}/constants.h
+        ${HERMES_ADAPTER_DIR}/singleton.h)
+set(PUBSUB_ADAPTER_SRC pubsub.cc metadata_manager.cc)
+
+# Add library hermes_stdio
```

Comment:
`hermes_stdio` -> `hermes_pubsub`. Or just remove the comment since it doesn't provide any information that's not already present in the line it comments on.